css/base.css

```diff
@@ -31,21 +26,29 @@
   padding: 2% 2% 2% 2%;
 }
 
+.socialMediaBtns a {
```

Comment:
There should only be element styling in `base.css`, all styling for smaller pieces, like for classes and ids, should be in `modules.css`.